pkg/server/cluster_server.go

```diff
@@ -33,36 +31,61 @@ const (
 var _ = Server(&clusterServer{})
 
 type clusterServer struct {
-	// machineClient is used to interact with the
-	// machine config, pool objects.
-	machineClient v1.MachineconfigurationV1Interface
+	machineConfigPoolLister v1.MachineConfigPoolLister
+	machineConfigLister     v1.MachineConfigLister
 
 	kubeconfigFunc kubeconfigFunc
 }
 
+const minResyncPeriod = 20 * time.Minute
+
+func resyncPeriod() func() time.Duration {
```

Comment:
You can use the `resyncPeriod` from [here](https://github.com/openshift/machine-config-operator/blob/605b94bfd58e9a9cde5643e2524177ecd2026841/pkg/controller/common/controller_context.go#L25)

Reply:
sorry, missed this. I intentionally didn't reuse it because the two components don't necessarily need to have the same resyncPeriod